packages/contracts/deployments/engine/V3/partners/gazelli-flex/DEPLOYMENTS.md

```diff
@@ -0,0 +1,86 @@
+
+# Deployment
+
+Date: 2023-08-11T21:50:20.616Z
+
+## **Network:** goerli
+
+## **Environment:** staging
+
+**Deployment Input File:** `deployments/engine/V3/partners/gazelli-flex/deployment-config.staging.ts`
+
+**GenArt721CoreV3_Engine_Flex:** https://goerli.etherscan.io/address/0x810Fe1794aD3Ab416E57933229AD3f784501E272#code
+
+**AdminACLV1:** https://goerli.etherscan.io/address/0xba1D8bB20f333aA12E6b24bDee4dBD217e574f19#code
+
+**Engine Registry:** https://goerli.etherscan.io/address/0xEa698596b6009A622C3eD00dD5a8b5d1CAE4fC36#code
+
+**MinterFilterV1:** https://goerli.etherscan.io/address/0xf04efd8Fbf8Ea83A2c8Dff327eC9F500335E0EB9#code
+
+**Minters:**
+
+**MinterSetPriceV4:** https://goerli.etherscan.io/address/0xe8fAFc6A4761f65B5AdCd9f52ee471F6746e0f65#code
+
+
+
+**Metadata**
+
+- **Starting Project Id:** 0
+- **Token Name:** GAZELL.iO
+- **Token Ticker:** GAZ
+- **Auto Approve Artist Split Proposals:** false
+- **Render Provider Address, Primary Sales:** deployer
+- **Platform Provider Address, Primary Sales:** 0xFf90F956C9A7d52aC21C37fe7a34801849887f19
+- **BytecodeStorageReader Library:** 0xB8B806A10d16cc80dB788552B54B3ECb4A2A3C3D
+
+**Other**
+
+- **Add initial project?:** false
+- **Add initial token?:** false
+- **Image Bucket:** gazell-io-goerli
+
+---
+
+
+# Deployment
+
+Date: 2023-08-11T21:54:13.805Z
+
+## **Network:** mainnet
+
+## **Environment:** mainnet
+
+**Deployment Input File:** `deployments/engine/V3/partners/gazelli-flex/deployment-config.mainnet.ts`
+
+**GenArt721CoreV3_Engine_Flex:** https://etherscan.io/address/0xA86cD4EceBd96085fCe4697614d30600803455C4#code
+
+**AdminACLV1:** https://etherscan.io/address/0xD47c1AEeeb3e59B32b671F15a258E6Ce26D1327f#code
+
+**Engine Registry:** https://etherscan.io/address/0x652490c8BB6e7ec3Fd798537D2F348D7904BBbc2#code
+
+**MinterFilterV1:** https://etherscan.io/address/0x0C35eb4b2646FF92E30cBB6f138006A7B74f7d60#code
+
+**Minters:**
+
+**MinterSetPriceV4:** https://etherscan.io/address/0x936130f3042EE825d375A3beeDA04A2bA4880197#code
+
+
+
+**Metadata**
+
+- **Starting Project Id:** 0
+- **Token Name:** GAZELL.iO
```

Comment:
quick check - is this the desired casing for token name? (all caps, except the `i`?)

Reply:
okay, looks legit based on quick google search. 👍

Reply:
Yezzzer. :)